java/integTest/src/test/java/glide/modules/VectorSearchTests.java

```diff
@@ -220,4 +230,376 @@ public void ft_drop() {
+                                "8.7",
+                                "votes",
+                                "1127635",
+                                "imdb_id",
+                                "tt0080684"))
+                .get();
+        client
+                .hset(
+                        prefixMovies + 11003,
+                        Map.of(
+                                "title",
+                                "The Godfather",
+                                "plot",
+                                "The aging patriarch of an organized crime dynasty transfers control of his"
+                                        + " clandestine empire to his reluctant son.",
+                                "release_year",
+                                "1972",
+                                "genre",
+                                "Drama",
+                                "rating",
+                                "9.2",
+                                "votes",
+                                "1563839",
+                                "imdb_id",
+                                "tt0068646"))
+                .get();
+        client
+                .hset(
+                        prefixMovies + 11004,
+                        Map.of(
+                                "title",
+                                "Heat",
+                                "plot",
+                                "A group of professional bank robbers start to feel the heat from police when they"
+                                        + " unknowingly leave a clue at their latest heist.",
+                                "release_year",
+                                "1995",
+                                "genre",
+                                "Thriller",
+                                "rating",
+                                "8.2",
+                                "votes",
+                                "559490",
+                                "imdb_id",
+                                "tt0113277"))
+                .get();
+        client
+                .hset(
+                        prefixMovies + 11005,
+                        Map.of(
+                                "title",
+                                "Star Wars: Episode VI - Return of the Jedi",
+                                "genre",
+                                "Action",
+                                "votes",
+                                "906260",
+                                "rating",
+                                "8.3",
+                                "release_year",
+                                "1983",
+                                "plot",
+                                "The Rebels dispatch to Endor to destroy the second Empire's Death Star.",
+                                "ibmdb_id",
+                                "tt0086190"))
+                .get();
+        Thread.sleep(1000); // let server digest the data and update index
```

Comment:
since we may have to add this sleep timer multiple times, want to make this a constant at the top of the test class?